client/webpack.storybook.config.js

```diff
@@ -37,6 +37,14 @@ config.module.loaders.push(
   }
 );
 
+config.externals = {
```

Comment:
This could be added as a comment here:  https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md